catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/CatalogMgrImplTest.java

```diff
@@ -22,9 +22,7 @@
 import static org.mockito.Mockito.when;
 
 import java.io.IOException;
-import java.util.ArrayList;
```

Comment:
These were unused.  The NSILI generated IDL does not adhere to checkstyle, so it has to be turned off.